app/autoload.php

```diff
@@ -1,29 +1,8 @@
 <?php
 
-/**
```

Comment:
All this file now does is require `vendor/autoload`. How about just removing it and requiring vendor/autoload directly where this file is required now.

Reply:
Good point, I'm removing the app/autoload.php and will use the composer autoload file directly instead.